android-json-form-wizard/src/test/java/com/vijay/jsonwizard/activities/JsonFormActivityTest.java

```diff
@@ -451,4 +459,161 @@ public void testGetRelevanceAddressReturnsExpectedAddressAndRelevancePair() thro
+                "        \"options\": [\n" +
+                "          {\n" +
+                "            \"key\": \"None\",\n" +
+                "            \"text\": \"None\",\n" +
+                "            \"value\": false,\n" +
+                "            \"openmrs_choice_id\": \"\"\n" +
+                "          },\n" +
+                "          {\n" +
+                "            \"key\": \"yes\",\n" +
+                "            \"text\": \"Yes\",\n" +
+                "            \"value\": false,\n" +
+                "            \"openmrs_choice_id\": \"\"\n" +
+                "          },\n" +
+                "          {\n" +
+                "            \"key\": \"no\",\n" +
+                "            \"text\": \"No\",\n" +
+                "            \"value\": false,\n" +
+                "            \"openmrs_choice_id\": \"\"\n" +
+                "          },\n" +
+                "          {\n" +
+                "            \"key\": \"other\",\n" +
+                "            \"text\": \"Other\",\n" +
+                "            \"value\": false,\n" +
+                "            \"openmrs_choice_id\": \"\"\n" +
+                "          }\n" +
+                "        ],\n" +
+                "        \"v_required\": {\n" +
+                "          \"value\": \"false\"\n" +
+                "        },\n" +
+                "        \"value\": \"[yes]\"\n" +
+                "      }";
+        Mockito.doReturn(new JSONObject(json)).when(view).getTag(R.id.json_object);
+        Whitebox.invokeMethod(activity, "updateUiByConstraints", view, true, errorMessage);
+        assertNotNull(view.getTag(R.id.key));
+    }
+
+    @Test
+    public void testUpdateUIByConstraintsOfRadioButton() throws Exception {
+        View view = Mockito.mock(RadioGroup.class);
+        String address = "test:key";
+        String childKey = "test";
+        String previous = "false";
+        String errorMessage = "false";
+        String key = "yes";
+        Mockito.doReturn(address).when(view).getTag(R.id.address);
+        Mockito.doReturn(childKey).when(view).getTag(R.id.childKey);
+        Mockito.doReturn(previous).when(view).getTag(R.id.previous);
+        Mockito.doReturn(key).when(view).getTag(R.id.key);
+        Mockito.doReturn("check_box").when(view).getTag(R.id.type);
+        String json = " {\n" +
+                "                \"key\": \"respiratory_exam_radio_button\",\n" +
+                "                \"type\": \"native_radio\",\n" +
+                "                \"values\": [\n" +
+                "                  \"3:Abnormal\"\n" +
+                "                ]\n" +
+                "              }";
+        Mockito.doReturn(new JSONObject(json)).when(view).getTag(R.id.json_object);
+        Whitebox.invokeMethod(activity, "updateUiByConstraints", view, true, errorMessage);
+        assertNotNull(view.getTag(R.id.key));
```

Comment:
Let's assert the specific value(s) and any other functionality eg broadcasts sent